template/ui/build/script.javascript.js

```diff
@@ -81,40 +81,40 @@ build(builds)
  * Helpers
  */
 
-function resolve (_path) {
-  return path.resolve(__dirname, _path)
-}
-
-function addAssets (builds, type, injectName) {
-  const
-    files = fs.readdirSync(resolve('../../ui/src/components/' + type)),
-    plugins = [ buble(bubleConfig) ],
-    outputDir = resolve(`../dist/${type}`)
-
-    fse.mkdirp(outputDir)
-
-  files
-    .filter(file => file.endsWith('.js'))
-    .forEach(file => {
-      const name = file.substr(0, file.length - 3).replace(/-([a-z])/g, g => g[1].toUpperCase())
-      builds.push({
-        rollup: {
-          input: {
-            input: resolve(`../src/components/${type}/${file}`),
-            plugins
-          },
-          output: {
-            file: addExtension(resolve(`../dist/${type}/${file}`), 'umd'),
-            format: 'umd',
-            name: `{{umdExportName}}.${injectName}.${name}`
-          }
-        },
-        build: {
-          minified: true
-        }
-      })
-    })
-}
+// function pathResolve (_path) {
+//   return path.resolve(__dirname, _path)
+// }
+
+// function addAssets (builds, type, injectName) {
```

Comment:
```suggestion
// eslint-disable-next-line no-unused-vars
function addAssets (builds, type, injectName) {
```

Disabling the rule for this line would be a better solution compared to commenting out ~30 lines.